torch/_dynamo/variables/user_defined.py

```diff
@@ -828,6 +832,7 @@ def var_getattr(self, tx, name):
             subobj = self._getattr_static(name)
         except AttributeError:
             subobj = NO_SUCH_SUBOBJ
+            log.warning(f"NO_SUCH_SUBOBJ: {self.value} {name}")
```

Comment:
More user-facing? e.g. "Attribute {name} does not exist on {self.value}, falling back to NO_SUCH_SUBOBJ"

Reply:
Can we also add the log of `self.value.__class__`?